### ISSUE TITLE
Option save or clone config

### ISSUE THREAD

**Issue by mac-linux-free** [Chrysostomus/aif-dev#203]
I'm looking for an option to save one config in a file and restore it on another PC

**Comment by Chrysostomus** [comment]
Do you mean saving your choices in the installer, or copying the running system into the installation, or something else?

**Comment by mac-linux-free** [comment]
yes saving all choices in the installer with partitioning and custom packages, security tweaks and users (passwords) to a file and having te file on an usb stick. on an other pc or vm I could then just launch the installer with this file.

would be great

**Comment by Chrysostomus** [comment]
This is planned feature, but not happening anytime soon. Possibly with 1.0 release or later. Currently there is such file, but it stores only language choices persistently and little else.

**Comment by Chrysostomus** [comment]
I've been looking into this, and I feel that it would be easiest to

1) expand the usage of the ini file 
2) write a separate script (manjaro-deploy?) that would use the ini file
3) possibly add an option to edit the ini file and call the deployment from manjaro-architect
